docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '2'
```

Comment:
No. This is the version of the docker-compose format, it has to be set to `2`. The older format was deprecated.